.readthedocs.yaml

```diff
@@ -9,7 +9,7 @@ build:
         - asdf install uv latest
         - asdf global uv latest
     install:
-        - cd bindings/python && uv sync --no-install-project --group docs
+        - cd bindings/python && uv sync --no-install-project --frozen --group docs
```

Comment:
WIth `frozen`.. what problems were popping up?

Reply:
When uv was updating the lock file it was trying to build the current project, but failed since libbson wasn't installed in the enironment.